updating signalr js package readme (#22129)
dotnet/aspnetcore: src/SignalR/clients/ts/signalr/README.md

JavaScript and TypeScript clients for SignalR for ASP.NET Core

> > Note: The JavaScript and TypeScript clients for SignalR for ASP.NET Core have been moved to [@microsoft/signalr](https://www.npmjs.com/package/@microsoft/signalr). If you are already using `@aspnet/signalr` and are unsure when to move to `@microsoft/signalr`, check the [Feature Distribution](https://docs.microsoft.com/en-us/aspnet/core/signalr/client-features) chart in the ASP.NET Core SignalR documentation.

Reviewer: I think putting the note on the very top is the way to go for this too.

Should we mention that we strive to keep "@microsoft/signalr" compatible with all older versions of ASP.NET Core SignalR? That should make it clear that developers can upgrade their clients ahead of their server(s).

Author: I'd welcome you adding something like that, or @scottaddie adding something. Thanks for the vote, Mr. Halter.

> > Note: The JavaScript and TypeScript clients for SignalR for ASP.NET Core have been moved to [@microsoft/signalr](https://www.npmjs.com/package/@microsoft/signalr). If you are already using `@aspnet/signalr` and are unsure when to move to `@microsoft/signalr`, check the [Feature Distribution](https://docs.microsoft.com/en-us/aspnet/core/signalr/client-features) chart in the ASP.NET Core SignalR documentation.

Reviewer 2: ```suggestion
> Note: The JavaScript and TypeScript clients for SignalR for ASP.NET Core have been moved to [@microsoft/signalr](https://www.npmjs.com/package/@microsoft/signalr). If you are already using `@aspnet/signalr` and are unsure when to move to `@microsoft/signalr`, check the [Feature Distribution](https://docs.microsoft.com/en-us/aspnet/core/signalr/client-features) chart in the ASP.NET Core SignalR documentation. Newer client releases are compatible with older version of ASP.NET Core SignalR which means it is safe to upgrade the client before upgrading the server.
```

> > Note: The JavaScript and TypeScript clients for SignalR for ASP.NET Core have been moved to [@microsoft/signalr](https://www.npmjs.com/package/@microsoft/signalr). If you are already using `@aspnet/signalr` and are unsure when to move to `@microsoft/signalr`, check the [Feature Distribution](https://docs.microsoft.com/en-us/aspnet/core/signalr/client-features) chart in the ASP.NET Core SignalR documentation.

Reviewer 3: ```suggestion
> Note: The JavaScript and TypeScript clients for ASP.NET Core SignalR have moved to [@microsoft/signalr](https://www.npmjs.com/package/@microsoft/signalr). If you're using `@aspnet/signalr` and are unsure when to move to `@microsoft/signalr`, check the [Feature distribution](https://docs.microsoft.com/aspnet/core/signalr/client-features) chart in the ASP.NET Core SignalR documentation.
```

Author: sgtm
